packages/flutter_tools/test/commands.shard/hermetic/precache_test.dart

```diff
@@ -38,159 +38,206 @@ void main() {
   });
 
   testUsingContext('precache should acquire lock', () async {
```

Comment:
can this be `testWithoutContext()`?

Reply:
All of the createTestCommandRunner tests hit the context due to FlutterCommand. I started refactoring things out, but I got stuck on output preferences and rolled it back a while ago